frontend/src/shared/video-card-image.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+
+interface Props {
+  videoId: string;
```

Comment:
I'm second thinking passing the videoId here in favor of the whole video, although we do _only_ use the `ID`, wdyt?

Reply:
Would be sensible, if we need other data from the video. Not sure whether that is needed though